vizro-core/CHANGELOG.md

```diff
@@ -11,6 +11,26 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
 
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.1.13'></a>
+
+# 0.1.13 — 2024-03-12
+
+### Highlights ✨
+
+- Introduce `AgGrid` as a new `Page` component, allowing the usage of
+  [AG Grid](https://www.ag-grid.com/javascript-data-grid/scrolling-scenarios/) in
```

Comment:
why link to the scrolling scenarios?

Reply:
Do we need to link to it at all? We don't do it for any of the other dash components e.g.

Would eventually just shorten to:

Introduce `AgGrid` as a new `Page` component. Visit the [user guide on tables](https://vizro.readthedocs.io/en/stable/pages/user_guides/table/) to learn more. ([#289](https://github.com/mckinsey/vizro/pull/289))